Header header
uint8 virtualwall
uint8 opcode_right
uint8 opcode_left
